toucan_connectors/databricks/databricks_connector.py

```diff
@@ -6,14 +6,17 @@
 import requests
 from pydantic import Field, SecretStr, constr
 from requests.auth import HTTPBasicAuth
-from werkzeug.exceptions import BadRequest
 
 from toucan_connectors.common import ConnectorStatus, pandas_read_sql
 from toucan_connectors.toucan_connector import ToucanConnector, ToucanDataSource
 
 logger = logging.getLogger(__name__)
 
 
+class DatabricksClusterException(Exception):
```

Comment:
Could you please create a base Exception class for all errors than can occur on cluster start ? Otherwise we'll have to handle every single one of them explicitly

Reply:
https://github.com/ToucanToco/toucan-connectors/pull/659/commits/06de879fc778a0de4365c1874f6b48bdaadabdc9